packages/terra-pills/src/private/PillsUtils.js

```diff
@@ -0,0 +1,49 @@
+const getPillWidth = (pill) => {
```

Comment:
We should rename this private file to `_PillsUtils.js` for consistency with the other private files.

Reply:
Renamed 51a9979